.travis.yml

```diff
@@ -14,19 +16,27 @@ stages:
     if: branch = master AND type = push
 
 .disable_global: &disable_global
+  addons: false
+  cache: false
+  env: {}
+  python: false
   before_install: false
-  install: true
+  install: false
   before_script: false
+  script: false
   after_success: false
   after_failure: false
+  before_deploy: false
+  deploy: false
 
-.check_moban: &check_moban
+.moban: &moban
   <<: *disable_global
   python: 3.6
   stage: moban
-  install: pip install moban
-  script: .misc/check_moban.sh
-  if: branch != master OR type = pull_request
+  install: pip install moban>=0.0.4
```

Comment:
Just a thought, In [test-requirements.txt](https://github.com/coala/coala/blob/master/test-requirements.txt#L15) we're installing another version of `moban` for python version > 3.0 and here we're installing a different version. Shouldn't they be same?